provider/core_internet_gateway_resource.go

```diff
@@ -171,6 +171,7 @@ func (s *InternetGatewayResourceCrud) SetData() {
 	s.D.Set("time_modified", s.Resource.ModifiedTime.String())
 	s.D.Set("state", s.Resource.State)
 	s.D.Set("time_created", s.Resource.TimeCreated.String())
+	s.D.Set("vcn_id", s.Resource.VcnID)
```

Comment:
Looks like this is missing in the data source as well.

Reply:
done